src/auto-evo/mutation_strategy/UpgradeOrganelle.cs

```diff
@@ -40,6 +57,12 @@ public UpgradeOrganelle(Func<OrganelleDefinition, bool> criteria, string upgrade
             return null;
         }
 
+        // If a cheaper organelle upgrade gets added, this will need to be updated
+        if (mp < 10)
+            return null;
+
```

Comment:
I think this code should be removed as this seems very unsafe as someone will add such an upgrade at some point, and this will be totally forgotten by then.

I'd allow this code *if* there's a check in SimulationParameters verifying on each game start that this is true.
I'd recommend just removing this so that such complex code doesn't need to be written.